packages/react/src/components/Button/Button.js

```diff
@@ -219,7 +219,7 @@ const Button = React.forwardRef(function Button(
       onMouseLeave: composeEventHandlers([onMouseLeave, handleMouseLeave]),
       onFocus: composeEventHandlers([onFocus, handleFocus]),
       onBlur: composeEventHandlers([onBlur, handleBlur]),
-      onClick: composeEventHandlers([handleClick, onClick]),
+      onClick: composeEventHandlers([onClick, handleClick]),
```

Comment:
Just out of curiosity what's the significance of the order change here? I haven't messed with the composeEventHandlers helper yet.

Reply:
nothing. It's just an array of helpers and I saw an opportunity to make it consistent with the others on*'s